platform/overlays/knative/images/dispatcher/dispatcher.py

```diff
@@ -0,0 +1,140 @@
+import os
+import sys
+import re
+import subprocess
+import json
+import logging
+import requests
+import jwt
+import threading
+from flask import Flask, request
+from datetime import datetime
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+app = Flask(__name__)
+
+hosts = ['https-scanner.tracker.example.com',
+         'ssl-scanner.tracker.example.com',
+         'dmarc-scanner.tracker.example.com']
+
+dkim_flagged_hosts = ['dkim-scanner.tracker.example.com,'
+                      'dmarc-scanner.tracker.example.com']
+
+manual_scan_hosts = ['https-scanner-manual.tracker.example.com',
+                     'ssl-scanner-manual.tracker.example.com',
+                     'dmarc-scanner-manual.tracker.example.com']
+
+manual_scan_dkim_flagged_hosts = ['dkim-scanner-manual.tracker.example.com,'
+                                  'dmarc-scanner-manual.tracker.example.com']
+
+
+@app.route('/receive', methods=['GET', 'POST'])
+def receive():
+
+    payload = {}
+    dkim_flag = False
+
+    try:
+        #TODO Replace secret
```

Comment:
Does this TODO need to be resolved?  If already done, consider removing comment.

Reply:
Resolved!